pyproject.toml

```diff
@@ -301,11 +301,7 @@ filterwarnings = [
 ]
 
 markers = ["slow: marks tests as slow (deselect with '-m \"not slow\"')"]
-# Oddly, despite the use of --cov-config=pyproject.toml here, pytest does not seem to
-# pick up the source directories specified in the [tool.coverage.run] section below.
-# (though it *does* pick up the omit parameters!). This means we need to specify the
-# source directories we want to collect coverage twice, and keep the two in sync.
-addopts = "--verbose --pdbcls=IPython.terminal.debugger:TerminalPdb --cov-append --cov-config=pyproject.toml --cov-report=xml --cov=src/pudl --cov=test/integration --cov=test/unit"
+addopts = "--verbose --pdbcls=IPython.terminal.debugger:TerminalPdb --cov-append --cov-config=pyproject.toml --cov-report=xml --cov -n auto"
```

Comment:
I don't remember exactly what was happening here, but it seemed to have to do with coverage behaving differently when run directly, vs. run through pytest.  But maybe I just didn't realize that a blanket `--cov` would default to whatever `sources` were listed in the configuration file. Seems likely.

We also need to deal with `-n auto` crashing your machine when you try to run the validation tests since we end up with potentially several copies of our largest dataframes getting read into memory simultaneously which is too much. But this problem isn't limited to this line.

Did you look at the `Makefile`, `.pre-commit-config.yaml` and `docker/gcp_pudl_etl.sh` to see if the pytest / coverage commands being used in there need to be updated to work with these `addopts` and/or avoid duplicative specifications of the same arguments in multiple places?